### PR TITLE
Omartin2010 patch 2

### PR DESCRIPTION
**Pull request by omartin2010**
fixed some of the documentation, removed a step from the pipeline.
Tagging @berndverst

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1801)

### CODE REVIEW

File: content/docs/azure/azureEndtoEnd.md [hunk 1]
```diff
@@ -78,6 +78,15 @@ To create a container registry:
     alt="Creating a Container Registry"
     class="mt-3 mb-3 p-3 border border-info rounded">
 
+### Allow your AKS Cluster access to your Azure Container Registry
+In order for the AKS cluster to have be able to pull images created for execution of the pipeline, you will need to update your cluster so that it is able to pull the images from the container registry we just created. More references can be found [here](https://docs.microsoft.com/en-us/azure/aks/cluster-container-registry-integration), on the Microsoft web site.
```
Comment (berndverst) [inline content/docs/azure/azureEndtoEnd.md]: > to have be able to pull
should be "to be able to pull" or "to have access to pulling"

**Comment by omartin2010** [comment]
fixed.

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1801--competent-brattain-de2d6d.netlify.com/docs/azure/azureendtoend/#allow-your-aks-cluster-access-to-your-azure-container-registry

/assign

**Comment by sarahmaddox** [comment]
Thanks @omartin2010 and @berndverst 

/ok-to-test
/lgtm
/approve

**Pull request merged**